modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/SettingsContainer/UserNotificationSettings.tsx

```diff
@@ -116,6 +116,12 @@ export function UserNotificationSettings({
 		setRolesList([]);
 	};
 
+	const getTerms = async () => {
+		const recipientList = values.recipients as UserNotificationRecipients[];
+
+		setToTerms(recipientList.map(({term}) => term).join());
+	};
+
```

Comment:
Hi @leilanyaragao, please, could you place `getTerms()` function before `getUserAccounts()`? Thanks! 🌷